jupyterlab_git/handlers.py

```diff
@@ -23,7 +23,7 @@
 from .log import get_logger
 
 # Git configuration options exposed through the REST API
-ALLOWED_OPTIONS = ["user.name", "user.email"]
+ALLOWED_OPTIONS = ["user.name", "user.email", "credential.helper"]
```

Comment:
Don't expose the helper to the frontend.

```suggestion
ALLOWED_OPTIONS = ["user.name", "user.email"]
```